mediaengine.go

```diff
@@ -0,0 +1,206 @@
+package webrtc
+
+import (
+	"strconv"
+
+	"github.com/pions/webrtc/internal/sdp"
+	"github.com/pions/webrtc/pkg/rtp"
+	"github.com/pions/webrtc/pkg/rtp/codecs"
+	"github.com/pkg/errors"
+)
+
+// PayloadTypes for the default codecs
+const (
+	PayloadTypeOpus = 111
+	PayloadTypeVP8  = 96
+	PayloadTypeVP9  = 98
+	PayloadTypeH264 = 100
+)
+
+// Names for the default codecs
+const (
+	Opus = "opus"
+	VP8  = "VP8"
+	VP9  = "VP9"
+	H264 = "H264"
+)
+
+var rtcMediaEngine = &mediaEngine{}
```

Comment:
Being a global, will this work if you have 2 separate PeerConnections that have conflicting PayloadTypes for different codecs?